python/ops.py

```diff
@@ -63,20 +87,132 @@ def c2r_ifft2(x_fft):
           tf.signal.ifft2d(tf.transpose(x_fft, [0, 3, 1, 2])), [0, 2, 3, 1]))
 
 
+def local_absolute_deviation(rgb):
+  """Compute a Local Absolute Deviation in sliding 3x3 window fashion.
+
+  This functions computes the edge signal of a given rgb image.
```

Comment:
Can you also add a reference to the eq. #19 in the paper?

Reply:
Added